lib/src/logic/validators.dart

```diff
@@ -416,4 +416,40 @@ class ShouldBeBetweenOrEqualValidator<KeyType>
 
   @override
   List<Object?> get props => [...super.props, min, max];
+}
+
+
+  extension DateTimeExtension on DateTime? {
+  
+  bool? isAfterOrEqualTo(DateTime dateTime) {
+    final date = this;
+    if (date != null) {
+      final isAtSameMomentAs = dateTime.isAtSameMomentAs(date);
+      return isAtSameMomentAs | date.isAfter(dateTime);
+    }
+    return null;
+  }
```

Comment:
Why do we need to handle null case here? We should extend on `DateTime` instead of `DateTime?`. The method `isValid` already uses non nullable date.

Also please use logical OR operator `||` instead of the bitwise operator `|`.

Same for other metohods.